examples/morpheusvm/cmd/morpheusvm/main.go

```diff
@@ -43,14 +43,18 @@ func main() {
 }
 
 func runFunc(*cobra.Command, []string) error {
-	if err := ulimit.Set(ulimit.DefaultFDLimit, logging.NoLog{}); err != nil {
-		return fmt.Errorf("%w: failed to set fd limit correctly", err)
-	}
+	logFactory := logging.NewFactory(logging.Config{
+		DisplayLevel: logging.Debug,
+	})
 
-	controller, err := controller.New()
+	log, err := logFactory.Make("main")
 	if err != nil {
-		return fmt.Errorf("failed to initialize controller: %w", err)
+		return fmt.Errorf("failed to initialize log: %w", err)
+	}
+
+	if err := ulimit.Set(ulimit.DefaultFDLimit, log); err != nil {
+		return fmt.Errorf("%w: failed to set fd limit correctly", err)
 	}
 
-	return rpcchainvm.Serve(context.TODO(), controller)
+	return rpcchainvm.Serve(context.TODO(), controller.New())
```

Comment:
Could we revert this diff? I think it's fine to pass `NoLog` into `ulimit.Set(...)` rather than create a log to pass in here.